GSA/usagov-benefits-eligibility
# #254 - Add critical application information field

## PR Summary

Issue #254 - Add critical application information field in SSA benefits.

## Related Github Issue

#254 

## Type of change

Please delete options that are not relevant.


## Branch Name

254-critical-informatioin

## Testing environment

https://federalist-edd11e6f-8be2-4dc2-a85e-1782e0bcb08e.sites.pages.cloud.gov/preview/gsa/usagov-benefits-eligibility/254-critical-information/death-of-a-loved-one/?

## Detailed Testing steps

Test remotely
1. go to https://federalist-edd11e6f-8be2-4dc2-a85e-1782e0bcb08e.sites.pages.cloud.gov/preview/gsa/usagov-benefits-eligibility/254-critical-information/death-of-a-loved-one/?
2. In benefit cards of SSA, can see critical application information "A single SSA benefit application will be used to apply to all SSA benefits, but eligibility and approval will be determined for individual benefits." in English and "Se utilizará una sola solicitud para solicitar todos los beneficios de la Administración del Seguro Social (SSA, por sus siglas en inglés), pero la elegibilidad y la aprobación de cada beneficio será determinada individualmente." in Spanish.

Test locally
1. pull this branch locally
2. run npm run dev in terminal
3. go to http://localhost:3000/death-of-a-loved-one/?
4. In benefit cards of SSA, can see critical application information "A single SSA benefit application will be used to apply to all SSA benefits, but eligibility and approval will be determined for individual benefits." in English and "Se utilizará una sola solicitud para solicitar todos los beneficios de la Administración del Seguro Social (SSA, por sus siglas en inglés), pero la elegibilidad y la aprobación de cada beneficio será determinada individualmente." in Spanish.

## Checklist:
